### PR TITLE
Refactored testsuite

### PR DESCRIPTION
**Pull request by mstruk**
This PR depends on #87

### CODE REVIEW

**Comment by mstruk** [comment]
Turns out too much logging results in test failure:

    The job exceeded the maximum log length, and has been terminated.

**Comment by scholzj** [comment]
Oh yeah, that is a lovely TravisCI feature :-o

File: .travis/build.sh [hunk 1]
```diff
@@ -38,19 +38,27 @@ mvn spotbugs:check
 # Run testsuite with java 8 only
 if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
 
-  docker pull oryd/hydra:v1.0.0
+  docker pull oryd/hydra:v1.8.5
+  docker pull quay.io/keycloak/keycloak:latest
```
Comment (scholzj) [inline .travis/build.sh:42]: Can we use some specific version instead of `:latest`?

File: .travis/build.sh [hunk 2]
```diff
@@ -38,19 +38,27 @@ mvn spotbugs:check
...
+
+  clearDockerEnv
+  docker pull quay.io/strimzi/kafka:latest-kafka-2.6.0
   mvn -e -V -B clean install -f testsuite -Pkafka-2_6_0
   EXIT=$?
   exitIfError
 
   clearDockerEnv
-  docker pull strimzi/kafka:latest-kafka-2.5.1
+  docker pull quay.io/strimzi/kafka:latest-kafka-2.5.1
```
Comment (scholzj) [inline .travis/build.sh:61]: I guess you can maybe use `0.21.1` instead of latest?

**Review by scholzj — APPROVED** [review]
I don't like the `latest` container images on so many places. Would prefer if it used some fixed version. But LGTM otherwise assuming the test pass of course.

File: .travis/build.sh (hunk 2 above)
Comment (mstruk) [reply .travis/build.sh:61]: Right, should make fore a more deterministic test env. Will change this.

File: .travis/build.sh (hunk 1 above)
Comment (mstruk) [reply .travis/build.sh:42]: Makes sense.

**Comment by mstruk** [comment]
I'm looking into the issue with the test failing ...

**Comment by mstruk** [comment]
> I'm looking into the issue with the test failing ...

Nevermind, local issue that I could only solve by doing `docker system prune`.

**Review by tombentley — APPROVED** [review]

**Pull request merged**